subekashi/static/subekashi/css/common.css

```
@@ -196,6 +237,21 @@ a {
     text-decoration: underline;
 }
 
+@media(min-height: 426px and max-height: 524px) {
```

`(が必要です`というエラーが発生しております。修正案は以下の通りです。
```suggestion
@media (min-height: 426px) and (max-height: 524px) {
```